resources/lib/services/msl/profiles.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 """MSL video profiles"""
 from __future__ import unicode_literals
-
+import xbmc
```

Comment:
Not sure it's used in the code

Reply:
You are right, it sould not be there, I used for debugging...

Reply:
You can use common.debug() instead ;)